requirements_dev.txt

```diff
@@ -0,0 +1,5 @@
+discretize
```

Comment:
This is a dev requirement?

Reply:
it is used in the tests. I guess we could install it only on travis, but I think if you are developing, you should have the ability to run all of the tests?